lighthouse-core/test/runner-test.js

```diff
@@ -44,9 +45,15 @@ describe('Runner', () => {
       }],
       audits: ['content-width'],
     });
+    const artifactsPath = '.tmp/test_artifacts';
+
+    after(() => {
+      const path = Runner._getArtifactsPath({auditMode: artifactsPath});
```

Comment:
test to verify that artifacts do exist at that location?  And preferably since `Runner._getArtifactsPath` is private no need to call it for rimraf if verified that runner is saving at that location